### PR TITLE
Formula override

### PR DESCRIPTION
**Pull request by mattjmeier**
Add an option to overwrite the DESeq2 formula via config file.

A few bugfixes added as well.

### CODE REVIEW

File: Rmd/Sample_QC.Rmd [hunk 1]
```diff
@@ -322,7 +322,8 @@ Summary of GC content of probes (TempO-Seq only)
 
 ggplot(count_data_gc,aes(x=GC_corrected_count,colour=sample)) +
   geom_density() +
-  scale_x_log10()
+  scale_x_log10() +
+  theme(legend.position = "none")
```
Comment (mattjmeier) [inline Rmd/Sample_QC.Rmd:326]: Labeling every sample is problematic with normal sized projects

File: config/config.default.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ common:
   dose: "dose"               # If there is a dose in the experiment; otherwise use NULL
   platform: "TempO-Seq"      # TempO-Seq Or RNA-Seq
   nmr_threshold: 100000    # 10% of 1M reads for TempOSeq = 100,000; 10% of 10M reads for RNA-Seq = 1,000,000.
-  write_additional_output: FALSE # export BMD, biomarker, and biosets output? facet variable should contain chemical and timepoint information
+  write_additional_output: TRUE # export BMD, biomarker, and biosets output? facet variable should contain chemical and timepoint information
```
Comment (mattjmeier) [inline config/config.default.yaml:12]: This should be the default option in my opinion - at the very least, to make sure code in there gets tested

File: scripts/file_functions.R [hunk 3]
```diff
@@ -85,26 +85,34 @@ save_cached_data <- function(dds, RDataPath, current_filter=NULL){
...
-    bmdexpress <- bmdexpress[rowSums(Counts) > 5,]
+    
+    if (params$platform == "TempO-Seq") {
+      biomarkers <- count_data
+      biomarkers$gene <- rownames(count_data)
+      biomarkers$gene <- gsub("_.*", "", biomarkers$gene)
+      biomarkers <- biomarkers %>%
+        dplyr::group_by(gene) %>%
+        dplyr::summarize(across(where(is.numeric), sum))
+      colnames(biomarkers)[[1]] <- "SampleID" # For reasons
```
Comment (cookkate) [inline scripts/file_functions.R:102]: lol

**Review by cookkate — APPROVED** [review]
Looks good

**Pull request merged**